bower.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Again, I'd suggest we not publish to bower as it's largely unmaintained.

Reply:
👍